### DISCUSSION TITLE
i9-11900K using UHD 750 does not support cl_khr_fp64

### DISCUSSION THREAD

**Discussion by knzivid** [intel/compute-runtime#517] [Q&A]
**Device name:** Intel(R) UHD Graphics 750 [0x4c8a]
**Driver version**: 22.09.22577

Is this a hardware limitation or will `cl_khr_fp64` be supported on this GPU in the future? I see it is disabled here

https://github.com/intel/compute-runtime/blob/4aef9925b0c6c6ade1a60aa3069ec77d6fff4a9a/shared/source/gen12lp/hw_info_rkl.cpp#L58

**Comment by nyanmisaka** [comment]
It's disabled by default since it's not native supported by hw but you can enable fp64 emulation by following this guide: https://github.com/intel/compute-runtime/blob/master/opencl/doc/FAQ.md#feature-double-precision-emulation-fp64
